src/schema.ts

```diff
@@ -265,6 +265,8 @@ export const typeDefs = gql`
       filterIds: String
     ): [CrossSubjectElement]
     oembed: String
+    conceptIds: [String]
+    concepts(conceptIds: [String]): [Concept]
```

Comment:
La til at article beholder den lista med conceptIds for å unngå å måtte sende den inn som parameter til resolveren for concepts. Ser jeg glemte å fjerne parameteren her, så skal få gjort det. Men er det best å gjøre det sånn her, eller hadde det vært bedre med en parameter?

Reply:
Bedre uten parameter som du har tenkt. Jeg tenker det blir tilsvarende competenceGoals og article.grepcodes over

Reply:
Bedre uten parameter som du har tenkt. Jeg tenker at det er tilsvarende competenceGoals og article.grepCodes over